src/main/java/com/sparta/popupstore/domain/point/dto/request/PointChargeRequestDto.java

```diff
@@ -0,0 +1,18 @@
+package com.sparta.popupstore.domain.point.dto.request;
+
+import com.sparta.popupstore.domain.point.entity.PointChargedLog;
+import com.sparta.popupstore.domain.user.entity.User;
+import lombok.Getter;
+
+@Getter
+public class PointChargeRequestDto {
+  private int chargedPoint;
+
+  public PointChargedLog toEntity(User user) {
```

Comment:
```suggestion
  public PointChargedLog toEntity(int point) {
```
user를 받지 말고 point 값만 받아오면 안되나요?